app/api/events.py

```diff
@@ -570,7 +558,7 @@ def get_id(view_kwargs):
             view_kwargs['id'] = None
 
     if view_kwargs.get('ticket_id') is not None:
-        ticket = safe_query(db, Ticket, 'id', view_kwargs['ticket_id'], 'ticket_id')
+        ticket = safe_query(Ticket, 'id', view_kwargs['ticket_id'], 'ticket_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)